lib/Unexpected.js

```diff
@@ -488,6 +488,8 @@ Unexpected.prototype.fail = function (arg) {
             var value = arg[key];
             if (key === 'diff') {
                 error.createDiff = value;
+            } else if (key === 'stack') {
+                error._stack = value;
```

Comment:
We could, but what would that accomplish, exactly? As far as I could tell we only need the `stack` trace of the original error (which includes the error message as well, in case we want that too).